pyproject.toml

```
@@ -25,12 +25,23 @@
   license = { file = "LICENSE" }
   requires-python = ">=3.9"
   readme = "README.md"
-  classifiers = ["License :: OSI Approved :: MIT License"]
+  classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Development Status :: 4 - Beta",
```

If we're marking this as version 1.0.0, really we should bump this now to:

```suggestion
    "Development Status :: 5 - Production/Stable",
```

I think having the current state marked as both version 1.0.0 and 'beta' would give out mixed messages and therefore be confusing.

---

Good point!